packages/pages-components/src/map/coordinate.ts

```diff
@@ -163,16 +163,16 @@ class Coordinate {
       longitude = typeof longitude == "function" ? longitude() : longitude;
     }
 
-    this.latitude = latitude;
-    this.longitude = longitude;
+    this._lat = latitude as number;
```

Comment:
Is the as number necessary?

Reply:
otherwise I get a ts error because latitude can be number | Object - is there a better way to fix that?

Reply:
Put it in an condition to check the typeof like above